init-tools.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
```

Comment:
we should use `set -e on this file so it will exit non-zero if any of the commands exit non-zero.